_site/docs/configuration/configuration.md

```diff
@@ -283,6 +283,24 @@ worker:
     execution: true
 ```
 
+### Sandbox Settings
+
+| Configuration | Accepted and _Default_ Values | Description                                     |
+|---------------|-------------------------------|-------------------------------------------------|
+| alwaysUse             | boolean, _false_      | Enforce that the sandbox be used on every acion.|
+| selectForBlockNetwork | boolean, _false_      | Only use the sandbox when blocking the network. |
+| selectForTmpFs        | boolean, _false_      | Only use the sandbox when using tmpfs.          |
```

Comment:
The Only language here confuses me: I'd change this to something like "block-network implies actions run under sandbox", if that's the right tag.